Sources/UtiliKit/Obfuscation/ObfuscatedKey.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  ObfuscatedKey.swift
+//  UtiliKit-iOS
+//
+//  Created by Russell Mirabelli on 7/23/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+// Inspired by https://github.com/onmyway133/Arcane/blob/master/Sources/Arcane/Obfuscator.swift
+
+import Foundation
+
+/// One should never commit keys directly into one's source; this is an unsafe practice.
+/// When it is impossible to avoid doing so, the key should at least be obfuscated.
+/// By using the ObfuscatedKey struct, you can build a human-readable key that nonetheless
+/// will not appear simply by running "strings" against your compiled code, and will even
+/// not appear as a string within your source code.
+struct ObfuscatedKey {
+    private let _value: String
+
+    init() {
+        self._value = ""
+    }
+
+    init(_ value: String) {
+        self._value = value
+    }
+
+    var value: String {
+        get {
+            return _value
```

Comment:
We can use the implicit getter here instead of manually specifying the `get` block.
```swift
var value: String {
    return _value
}
```

Reply:
done